spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -310,7 +310,7 @@ public void testRewriteToSmallerTarget() throws Exception {
             .option(SizeBasedFileRewriter.TARGET_FILE_SIZE_BYTES, String.valueOf(avgSize / 2))
             .execute();
     List<DeleteFile> newDeleteFiles = deleteFiles(table);
-    assertThat(newDeleteFiles).as("Should have 8 new delete files").hasSize(8);
+    assertThat(newDeleteFiles).as("Should have new delete files").hasSize(8);
```

Comment:
nit: Shall we unify the message and use `Expected new delete file(s)` as in line 195?

Reply:
Updated